build.xml

```diff
@@ -20,6 +20,57 @@
     </target>
 -->
 
+  <scriptdef name="scepcificTest" language="javascript">
+    <attribute name="cmcuser"/>
+    <attribute name="cmcpass"/>
+    <attribute name="serverurl"/>
+    <![CDATA[
+    	//*** CREATE DEPLOY TASK ****
+		var deployTask = project.createTask('antlib:com.salesforce:deploy');
+		var username = attributes.get("cmcuser");
+		var password = attributes.get("cmcpass");
+		var serverurl = attributes.get("serverurl");
+		var source = project.getProperty("source");
+		var version = project.getProperty("version");
+		var specifiedTest = project.getProperty("specifiedTests");
+		
+		if(username!=null && password!=null && serverurl!=null && source!=null && version!=null && specifiedTest!=null){
+			deployTask.setUsername(username);
+			deployTask.setPassword(password);
+    	    deployTask.setServerURL(serverurl);
+    	    deployTask.setDeployRoot(source + " sprint " + version);
+    	    deployTask.setTestLevel("RunSpecifiedTests");
+    	    deployTask.setMaxPoll("5000");

```

Comment:
Minor formatting issue, but looks like you have some spacing/tabbing issues between lines 37-44.